README.md

```diff
@@ -6,11 +6,11 @@ Hi,
 
 This is a training repository for Kottans [frontend](https://github.com/Kottans/frontend),
 [backend](https://github.com/Kottans/backend),
-and [git](https://github.com/Kottans/git-course) courses. 
+and [git](https://github.com/Kottans/git-course) courses.
```

Comment:
There are a few things that need to be improved. There is no need to remove spaces where text is added by other students. This applies to this place and to similar ones below in the document.